### PR TITLE
Makefile: various fixes

### PR DESCRIPTION
**Pull request by joelanford**
- explicitly configure shell
- change CONTAINER_RUNTIME error to warning
- fix broken GORELEASER_ARGS setup

### CODE REVIEW

**Comment by joelanford** [comment]
/hold
Trying to convince myself the changes will actually result in the publishing step happening if we cut a new tag after this merges.

**Comment by joelanford** [comment]
I think I found the issue with the 1.21 release GHA. I messed up the GORELEASER_ARGS variable setup in a previous PR.

I think this is likely to fix the issue
/hold cancel

**Review by rashmigottipati — APPROVED** [review]
/lgtm

**Review by grokspawn — APPROVED** [review]

**Pull request merged**
